client/js/nonprofits/donate/get-params.js

```diff
@@ -18,6 +18,9 @@ module.exports = params => {
       const [name, label] = R.map(R.trim, R.split(':', f))
       return {name, label: label ? label : name}
     }, R.split(',',  fields))
+  , tags: tags => R.map(tag => {
+      return tag.trim()
+    }, R.split(',', tags))
```

Comment:
What happens if the  format of `data-tags` is nothing like what we expect? I don't see how this would fail but...

Reply:
It doesn't fail in the cases I tested, is there any particular case that you think it would be good to check?

Reply:
I honestly doubt there are are any but you never know!